src/vic-stdlib.c

```diff
@@ -44,6 +46,33 @@ void vic_get(void)
     }
 }
 
+void vic_list(void)
+{
+    char option = '\0';
+    char *args_format = "-%c ";
+
+    if (vic_args(args_format, &option) != 1) {
```

Comment:
@Adman I believe that is questionable as it in the end boils down to what should `ls` actually list.

The other question also is whether we want to have an option for `ls` or rather a whole new command (like ls-var).